src/test/java/core/basesyntax/writereport/WriteReportImplTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.writereport;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+class WriteReportImplTest {
+
+    private final WriteReportImpl writeReport = new WriteReportImpl();
+
+    @Test
+    void writeToFile_validInput_fileContainsContent(@TempDir Path tempDir) throws IOException {
+        String content = "Test content";
+        Path filePath = tempDir.resolve("testFile.txt");
+        writeReport.writeToFile(content, filePath.toString());
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath.toFile()))) {
+            String line = reader.readLine();
+            assertEquals(content, line, "File content does not match the written content");
+        }
+    }
+
+    @Test
+    void writeToFile_invalidPath_throwsRuntimeException() {
```

Comment:
```suggestion
    void writeToFile_invalidPath_notOk() {
```
use the same name pattern